### PR TITLE
Support for link-to-file configuration option

### PR DESCRIPTION
**Pull request by Cypher1**
Fixes https://github.com/dylanowen/mdbook-graphviz/issues/16

### CODE REVIEW

**Review by dylanowen — APPROVED** [review]
Great stuff 👍

File: src/renderer.rs [hunk 1]
```diff
@@ -10,9 +10,17 @@ use regex::Regex;
...
+pub struct GraphvizRendererConfig {
+    pub link_to_file: bool,
+}
+
 #[async_trait]
 pub trait GraphvizRenderer {
-    async fn render_graphviz<'a>(block: GraphvizBlock) -> Result<Vec<Event<'a>>>;
+    async fn render_graphviz<'a>(
+        block: GraphvizBlock,
+        _config: &GraphvizRendererConfig,
```
Comment (dylanowen) [inline src/renderer.rs:22]: nit: I don't believe we need the `_` for config here

**Comment by Cypher1** [comment]
Closed in https://github.com/dylanowen/mdbook-graphviz/pull/119

File: src/renderer.rs (hunk 1 above)
Comment (Cypher1) [reply src/renderer.rs:22]: Ah, forgot to make this change. I'll have to get it next time.

**Pull request closed without merging**